docs/3.0/vsTestAdapter.html

```diff
@@ -0,0 +1,112 @@
+<h2>NUnit 3.0 Test Adapter for Visual Studio</h2>
+
+<p>The NUnit 3.0 Test Adapter allows you to run NUnit 3.0 tests inside Visual Studio. This is a new adapter, based partly on the code of the original NUnit Test Adapter, but modified to work with NUnit 3.0.</p>
+
+<p>It is not possible to run NUnit 2.x tests using this adapter. Use the original adapter for that purpose. If you need to work with projects using NUnit 2.x and other projects using NUnit 3.0, you may install both versions of the adapter.</p>
+
+<p>The current 3.0 Beta release is designed to work with Studio 2012 (All updates), and with Visual Studio 2013 (All updates including Update 3 checked September 2014). Some features are not available under VS2012 RTM.</p>

```

Comment:
I agree except for dropping the last sentence. That's still true although few folks should be running it.